### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 10)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

**Review by msewaweru — APPROVED** [review]
LGTM!

**Review by cilwerner — APPROVED** [review]
@msewaweru, this looks good, can be merged.

File: azureadps-2.0-preview/AzureAD/Set-AzureADMSConditionalAccessPolicy.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ Conditional access policies are custom rules that define an access scenario.
 
 ### Example 1: Updates a conditional access policy in Azure AD by PolicyId.
 ```
-PS C:\> Set-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358 -DisplayName "MFA policy 1" -State "Enabled"
+PS C:\> Set-AzureADMSConditionalAccessPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5 -DisplayName "MFA policy 1" -State "Enabled"
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Set-AzureADMSConditionalAccessPolicy.md:30]:
```suggestion
PS C:\> Set-AzureADMSConditionalAccessPolicy -PolicyId 6b5e999b-0ba8-4186-a106-e0296c1c4358 -DisplayName "MFA policy 1" -State "Enabled"
```

File: azureadps-2.0-preview/AzureAD/Set-AzureADMSConditionalAccessPolicy.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ Conditional access policies are custom rules that define an access scenario.
...
 
-          Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
+          Id                      : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0-preview/AzureAD/Set-AzureADMSConditionalAccessPolicy.md:32]:
```suggestion
          Id                      : 6b5e999b-0ba8-4186-a106-e0296c1c4358
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, only one revert needed here.

**Review by msewaweru — APPROVED** [review]
Good to go, @CelesteDG.

**Pull request merged**